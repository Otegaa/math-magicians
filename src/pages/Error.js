import React from 'react';

const Error = () => (
  <div className="error">
    <h2>404!</h2>
    <p>Page not found</p>
  </div>
);

export default Error;
