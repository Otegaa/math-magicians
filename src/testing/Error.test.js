// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from 'react-test-renderer';
import Error from '../pages/Error';

it('renders correctly', () => {
  const tree = renderer.create(<Error />).toJSON();
  expect(tree).toMatchSnapshot();
});
