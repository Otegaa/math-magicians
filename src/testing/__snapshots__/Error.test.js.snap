// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  style={
    Object {
      "fontSize": "2rem",
      "marginTop": "20rem",
    }
  }
>
  <h2>
    404!
  </h2>
  <p>
    Page not found
  </p>
</section>
`;
